collect_app/src/main/res/values/strings.xml

```diff
@@ -514,4 +514,5 @@
     <string name="form_delete_message">Deleting selected forms</string>
     <string name="deleting_form_dialog_update_message">Deleting form: %1$d out of %2$d </string>
     <string name="search">Search</string>
+    <string name="null_intent_value">Returned intent has a null value but it is required.</string>
```

Comment:
This is going to be confusing to user and also really hard for translators to work with! How about something like "The external application did not provide expected information." Same concept but a little more human friendly. 😊